src/govuk/components/footer/footer.yaml

```diff
@@ -28,7 +28,7 @@ params:
     - name: href
       type: string
       required: true
-      description: List item href attribute in the meta section of the footer.
+      description: List item's `href` attribute in the meta section of the footer.
```

Comment:
Agree with the change, but the wording here still seems a bit strange as the `href` attribute goes on the link not the list item.

Most of the descriptions in this section could be improved though, so maybe we should re-visit later?

Reply:
That's a fair point. Yeah, that sounds good.

Reply:
```suggestion
      description: List item href attribute in the meta section of the footer.
```